.github/workflows/test.yml

```diff
@@ -15,9 +15,8 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: '14'
+          node-version: '18.x'
```

Comment:
consider `latest`. the build tools will always work on LTS releases.
https://github.com/actions/setup-node#supported-version-syntax